util/reggen/gen_dv.py

```diff
@@ -72,6 +75,63 @@ def gen_core_file(outdir: str,
         yaml.dump(core_data, core_file, encoding='utf-8')
 
 
+def get_all_dv_base_names(dv_base_prefix: str) -> Dict[str, List[str]]:
+    ''' Return all DV base names in a dictionary.
+
+    The output format is:
+    {
+      block_1: ["base_reg_block_name", "base_reg_name", "base_reg_field_name", "base_reg_pkg"],
```

Comment:
use Dict for the list of base class name, otherwise, [0] is not very readable
```suggestion
      block_1: [reg_block:"base_reg_block_name", reg:"base_reg_name", reg_field:"base_reg_field_name", reg_pkg:"base_reg_pkg"],
```

Reply:
Thank you! That is a good idea. Yeah initially I though about using an enum, and was not that helpful. I will update that.